presentation/presentation.slide

```diff
@@ -0,0 +1,205 @@
+Welcome to go-centrifuge
+
+
+An overview of architecture and evolution
+Tags: golang, go-centrifuge, supply-chain-financing, fintech
+
+
+_Brought_ _to_ _you_ by *go-centrifuge* contributors
+
+https://www.centrifuge.io
+https://developer.centrifuge.io
+
+
+* Intro
+
+*go-centrifuge* is the go implementation of the Centrifuge OS interacting with the peer to peer network and our Ethereum smart contracts.
+
+
+* What is this presentation about?
+
+- An overview of how *go-centrifuge* works
+- It documents the current state of *go-centrifuge* architecture
+- It helps in driving discussion where we want to _evolve_ the architecture
+- An architecture checkpoint in git so that we can understand our progress through time
+
+
+* What this presentation is NOT about?
+
+- A deep dive into specific functionality(eg: precise-proofs or transactions) or tech in the Node (But it would provide pointers on how to find those details)
+- Fully fledged documentation on how to run a Node.
+.link https://developer.centrifuge.io visit developer documentation for that.
+
+* Part 1 - Current State
+
+* A High Level View
+
+  +-----+----+--------------------------------+
+  |     |    |     Storage (leveldb)          |
+  |     |    |                                |
+  |     |    +--------------------------------+
+  |     |    |    Repository                  |
+  |     |Log |                      +-------+ |
+  |     |    +----------------------+ Model +-------+----+
+  |     |    |      Service         |       | |     |    |
+  |     |    |                      |   +---+ |     |    |
+  |Config    +----------------------+   |CD +-+ TX  |ETH |
+  |     |    |                      +-------+ |     |    |
+  |     |    | Controller (Handler)           |     |    |
+  |     |    +---------------+----------------------+----+
+  |     |    |  API          |   P2P          |Queue     |
+  |     |    |               |                |          |
+  +-----+----+---------------+----------------+----------+
+  |                       Node                           |
+  |                                                      |
+  +------------------------------------------------------+
+
+
+* Wait but first about Errors :)
+
+We have a few conventions around error handling,
+- All errors should be checked and propagated up the call chain (no panic or fatal)
+- Use constant error values
+- Use *github.com/centrifuge/go-centrifuge/errors* package to create errors
+
+* Bootstrappers
+
+.code ../bootstrap/bootstrapper.go /Bootstrapper/,/^}/
```

Comment:
@vedhavyas this is an example of a code reference

Reply:
You can always refer this from another repo AFAIK

Reply:
I checked, it wouldn't include the code blocks with another repo link. Anyway these slides are intended as documentation, Its not a presentation per se. So I think we can just keep it in the repo it self. Its also easier to keep it update with code changes this way.